bgpd/bgp_vty.c

```diff
@@ -5196,9 +5196,17 @@ DEFUN (no_neighbor_peer_group,
 	VTY_DECLVAR_CONTEXT(bgp, bgp);
 	int idx_word = 2;
 	struct peer_group *group;
-
+	afi_t afi;
 	group = peer_group_lookup(bgp, argv[idx_word]->arg);
 	if (group) {
+		for (afi = AFI_IP; afi < AFI_MAX; afi++) {
+			int lr_count = listcount(group->listen_range[afi]);
+			if (lr_count) {
+				vty_out(vty, "%% Cannot delete peer-group. Currently attached to %d listen-range(s). Delete listen-range(s) first\n",
```

Comment:
IMO this is enough to print like:

```
Peer-group %s is attached to %d listen-range(s), delete them first\n
```

Reply:
Updated Log message as per suggestion.
Thanks